com.github.elework.spreadsheet.yml

```
@@ -0,0 +1,18 @@
+app-id: com.github.elework.spreadsheet
+runtime: io.elementary.Platform
+runtime-version: 'daily'
+sdk: io.elementary.Sdk
+command: com.github.elework.spreadsheet
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --filesystem=xdg-documents
```

```suggestion
  - --filesystem=home
```

While we use the user's Documents folder as a default document folder, many user may want to open their files not in that folder. So I feel we may want to grant access to the use's Home directory at least

---

It's stated in the flatpak docs and implied by elementary that you shouldn't expose the home and system folders unnecessarily since it is complete overkill. If a user wants to expose a specific folder outside Documents, they can do that by selecting it when prompted to open a file using the filechooser portal (which needs implementing since I recall only seeing the old gtk filechooser)

---

Ah, indeed, you're right! Let's keep it as you write then :+1: